src/createElement.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import StemContainer from './StemContainer';
```

Comment:
Agreed. It just kinda sucks that this project doesn't have a good unique name and consists of three buzzwords `react-router-relay`, or `RRR` :D